this is already handled by the unsigned overload on 32bit

--- a/game/utils/string_frm.h
+++ b/game/utils/string_frm.h
@@ -135,11 +135,13 @@ class alignas(64) string_frm {
       implWrite(out, maxSz, at, buf);
       }
 
+#if __SIZEOF_SIZE_T__ == 8
     void implWrite(char* out, size_t maxSz, size_t& at, size_t arg) {
       char buf[20] = {};
       std::snprintf(buf,sizeof(buf),"%u",uint32_t(arg));
       implWrite(out, maxSz, at, buf);
       }
+#endif
 
     void implWrite(char* out, size_t maxSz, size_t& at, float arg) {
       char buf[20] = {};
